### ISSUE TITLE
increased/decreased circulating insulin AUC

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3582]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

A. increased circulating insulin AUC
B. decreased circulating insulin AUC

>2. Synonyms (e.g., absent spleen)

A. increased circulating insulin area under the curve
B. decreased circulating insulin area under the curve

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

A. greater than the normal concentration of insulin in the blood over a specified time of measurement
B. less than the normal concentration of insulin in the blood over a specified time of measurement

PMID:10323602

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

A. increased circulating insulin level (MP:0002079)
B. decreased circulating insulin level (MP:0002727)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
I'm not inclined to add this term as it is very measurement specific. @slaulederkind do you have an idea of how this is distinct from increased/decreased insulin level other than it is the are under the curve?

**Comment by slaulederkind** [comment]
No, it is just a more granular description of the measurement.

**Comment by sbello** [comment]
Okay, I think then we don't want to add this to the MP. While we have some measurement specific terms these are generally where the measurement is the phenotype.
